.env.example

```diff
@@ -0,0 +1,2 @@
+PARALLEL_RUN=
```

Comment:
Didn't we remove this feature?

Reply:
![image](https://github.com/TypedDevs/bashunit/assets/6353105/238903fc-d2af-4ee6-88b1-70e19d0f8a7f)

I would say nop, but I can erase it